apps/workspaces/actions.py

```
@@ -18,7 +19,7 @@
 logger.level = logging.INFO
 
 
-def export_to_intacct(workspace_id: int, export_mode: bool = None, expense_group_ids: list = []) -> None:
+def export_to_intacct(workspace_id: int, export_mode: bool = None, expense_group_ids: list = [], triggered_by: ExpenseImportSourceEnum = None) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)